cumulus/ansible/tasks/providers/ec2.py

```diff
@@ -187,6 +187,15 @@ def get_volume(self, girder_volume):
 
         return volume
 
+    def get_machine_images(self, name, owner):
+        filters = [{
+            'Name': 'name',
+            'Values': [name]
+        }]
+        r = self.client.describe_images(Owners=[owner], Filters=filters)
+
+        return [{'image_id': image['ImageId']} for image in r['Images']]
```

Comment:
@kotfic I used underscores here to be consistent with the rest of the class. However, this seems to do against the camel case convention?